### ISSUE TITLE
Evaluating the transfer-learning model

### ISSUE THREAD

**Issue by metouitude** [clemsage/NeuralDocumentClassification#1]
![Screenshot from 2021-08-04 05-41-43](https://user-images.githubusercontent.com/29012045/128118828-99a30d76-ea8e-4e77-80cb-075ab8b511e7.png)
Hey thank you for uploading your project this is really helpful.
I'm trying to evaluate the model built with transfer learning, looks like there is a shape issue..

**Comment by clemsage** [comment]
Hi Mohamed,

If I understand you correctly, you want to evaluate a convolutional model that was initialized by Inceptionv3, after having trained it with the last cell of the notebook skeleton.ipynb, is that correct ? It seems that the images of the test set that were originally resized at 385x297x1 for the non pre-trained models are not resized to the size expected by such a pre-trained model (299x299x3). 

Have you modified the test dataset (labeled_test_ds) as we have modified the train dataset (labeled_train_ds) (see below) ?

![image](https://user-images.githubusercontent.com/15333156/128147949-0d3038c0-ee96-450e-bcb7-6368bc252ffa.png)

**Comment by metouitude** [comment]
Hey! thanks for the reply and yes you got it exactly right.
I modified the test dataset and now it's evaluating! solved
![Screenshot from 2021-08-04 15-18-39](https://user-images.githubusercontent.com/29012045/128187846-260d3179-3d3a-4575-b20b-6c15502452c5.png)

I wish you all the best in the world !
